apps/server/.eslintrc.json

```diff
@@ -2,5 +2,9 @@
     "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
     "parser": "@typescript-eslint/parser",
     "plugins": ["@typescript-eslint"],
-    "root": true
-}
\ No newline at end of file
+    "root": true,
+    "rules": {
+        "@typescript-eslint/no-namespace": "off",
+        "@typescript-eslint/no-explicit-any": "off"
```

Comment:
disabling no explicit any rule completely might not be the best idea. If you need it disable it in the line or in the file. You can even disable it in the files which has a specific pattern like `**.test.ts`. In essence no-any usage should be the general rule, disabling it should be the exception.